### ISSUE TITLE
[NUnit1031] False positive for nullable structs

### ISSUE THREAD

**Issue by MaceWindu** [nunit/nunit.analyzers#938]
Version: 4.11.0

```cs
The value of the argument at position '0' of type SomeEnum cannot be assigned to the parameter 'type' of type SomeEnum? (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit1031.md)
The value of the argument at position '1' of type object cannot be assigned to the parameter 'type' of type SomeEnum? (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit1031.md)
public void TestMethod([Values(SomeEnum.Value, null)] SomeEnum? value)
{
}
```

**Comment by mikkelbu** [comment]
Thanks for the report @MaceWindu. It seems like @maksim-sovkov have already fixed the problem
